apps/src/templates/SchoolDataInputs.jsx

```diff
@@ -43,35 +43,35 @@ export default function SchoolDataInputs({
 
   return (
     <div className={style.outerContainer}>
-      <Heading2>{i18n.censusHeading()}</Heading2>
-      <BodyOneText>{i18n.findYourSchool()}</BodyOneText>
+      <Heading2 className={style.topPadding}>{i18n.censusHeading()}</Heading2>
+      <BodyTwoText>{i18n.findYourSchool()}</BodyTwoText>
       <div className={style.inputContainer}>
-        <Heading6 className={style.topPadding}>{i18n.whatCountry()}</Heading6>
+        <Heading6 className={style.padding}>{i18n.whatCountry()}</Heading6>
```

Comment:
I have one other feedback for the label elements that I thought about this morning! Let's use the `<BodyTwoText>` component here (and in place of the other Heading6) with the `visualAppearance={'heading-sm'}` prop. 

This is better for accessibility since these aren't h6 headings, and I think that might solve some of the styling issues. If it looks weird after doing this let me know if you want to pair again!

Reply:
The heading-sm looks too big, so I went with heading-xs instead. Is that alright?

Reply:
I think it should be fine! Can you post a screenshot here or in the top description area to show the updates? 🙏

Reply:
Oh sure! It looked the exact same as the Heading6 so I didn't take a pic initially. Here it is:
<img width="697" alt="Screenshot 2024-02-29 at 4 19 09 PM" src="https://github.com/code-dot-org/code-dot-org/assets/37230822/e8eeab2a-3aa2-4aaa-95ff-94653864e8e6">

Reply:
Uuuuugh but now there's a difference in padding above the input box (.25rem) vs above the dopdown boxes🤦🏻‍♀️ (.37rem)

Reply:
That looks great! When you update [this](https://github.com/code-dot-org/code-dot-org/pull/56889#discussion_r1508338557) it should be good to go.